drivers/i3c/Kconfig

```diff
@@ -0,0 +1,95 @@
+# I3C configuration options
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig I3C
+	bool "I3C Drivers"
```

Comment:
Shouldn't this also `select I2C`. I see the the `i3c.h` includes the `i2c.h` header.

Reply:
This is just for the struct, and should not be forcing I2C as it is possible to have a pure I3C bus with no I2C devices. A further optimization is to disable any I2C related functionalities.